server/services/sessionTemplateService.ts

```diff
@@ -89,14 +90,14 @@ export default class SessionTemplateService {
     return sessionTemplate
   }
 
-  async getTemplateStats(
-    username: string,
-    requestVisitStatsDto: RequestSessionTemplateVisitStatsDto,
-    reference: string,
-  ): Promise<SessionTemplateVisitStatsDto> {
+  async getFutureTemplateStats(username: string, reference: string): Promise<SessionTemplateVisitStatsDto> {
```

Comment:
Refactoring this service method to just get future booked visits to avoid duplication in the controllers because (currently) we only ever request future booked visits from today.